api/bases/watcher.openstack.org_watcherapis.yaml

```diff
@@ -65,16 +72,86 @@ spec:
                 description: PreserveJobs - do not delete jobs after they finished
                   e.g. to check logs
                 type: boolean
+              replicas:
+                default: 1
+                description: Replicas of Watcher service to run
+                format: int32
+                maximum: 32
+                minimum: 0
+                type: integer
+              resources:
+                description: |-
+                  Resources - Compute Resources required by this service (Limits/Requests).
+                  https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+                properties:
+                  claims:
+                    description: |-
+                      Claims lists the names of resources, defined in spec.resourceClaims,
+                      that are used by this container.
+
```

Comment:
there are a few empty lines in the crds please check (seems to be in the same places)